Change the chainandblocks_ceiling tilecode's image to one that makes more sense.

jaythebusinessgoose opened the pull request:
The new image is the chain_ceiling image with a small image of one of the chainblocks in the bottom right corner.

jaythebusinessgoose commented on src/modlunky2/sprites/tilecode_extras.py:
```diff
@@ -93,7 +93,8 @@ class TilecodeExtras(BaseSpriteLoader):
         tilename: (idx, 0, idx + 1, 1) for idx, tilename in enumerate(TILENAMES)
     }
-
+    _chain_index = TILENAMES.index("chain_ceiling")
```
I am not sure about this part, is this an acceptable way of making a temp var for this purpose?

gmjosack replied:
I don't really like the overrides like this. maybe update the `"chain_ceiling"` entry to be `["chain_ceiling", "chainandblocks_ceiling"]`. and add a

```py
def _generate_chunk_map():
    chunk_map = {}
    for idx, tilenames in enumerate(TILENAMES):
        if isinstance(tilenames, str):
            tilenames = [tilenames]
        for tilename in tilenames:
            chunk_map[tilename] = (idx, 0, idx+1, 1)
    return chunk_map
```
this lets us easily reuse any index for multiple tile names.

jaythebusinessgoose replied:
That is a good idea, however I decided to just create a new image for the chainandblocks_ceiling instead since that way it can be differentiated from the chain_ceiling.

gmjosack commented:
I was planning to move away from using this file so every individual change doesn't bloat the repo with another copy of the binary file. This is an example commit where i added one new image https://github.com/spelunky-fyi/modlunky2/commit/cf9672c0145bc1f13a44ba1e2cd72d9786fb322b

jaythebusinessgoose commented:
> I was planning to move away from using this file so every individual change doesn't bloat the repo with another copy of the binary file. This is an example commit where i added one new image [cf9672c](https://github.com/spelunky-fyi/modlunky2/commit/cf9672c0145bc1f13a44ba1e2cd72d9786fb322b)

Haha, funnily enough I had it this way originally then thought that we wouldn't want a separate image+class for each tilecode. Reverted back to where I was doing this, and I think it's the same as the vault chests now.

gmjosack commented:
Yea i'll probably make some helper to simplify making these eventually but rather have that then constantly changing the giant image file
